tests/ios/xcodeproj/BUILD.bazel

```diff
@@ -4,6 +4,13 @@ xcodeproj(
     name = "Single-Static-Framework-Project",
     testonly = True,
     bazel_path = "bazelisk",
+    custom_build_options_by_config = {
```

Comment:
I wonder if some options should always be there for `debug`/`release`? Is there a use case where `-c opt` for `release` should not be there for example?

Also, maybe the attr name should have the word `bazel` in it to be explicit about what custom options can be added here? Just a thought.

Reply:
>> some options should always be there for debug/release 
Probably, but it's up to the developer to decide what thy want. The library probably should be agnostic to that.

>> maybe the attr name should have the word bazel
Thought about that, but decided against that because it's a bazel rule. No need to duplicate it again.